src/components/v2/V2Create/forms/TokenForm/index.tsx

```diff
@@ -42,13 +42,29 @@ import SwitchHeading from 'components/shared/SwitchHeading'
 import NumberSlider from 'components/shared/inputs/NumberSlider'
 
 import FormItemWarningText from 'components/shared/FormItemWarningText'
+import { formattedNum } from 'utils/formatNumber'
 
 import { shadowCard } from 'constants/styles/shadowCard'
 import TabDescription from '../../TabDescription'
 
 const MAX_DISCOUNT_RATE = 20 // this is an opinionated limit
+export const DEFAULT_ISSUANCE_RATE = 10 ** 6
```

Comment:
Lets make this a constant somewhere and reuse it here and in https://github.com/jbx-protocol/juice-interface/blob/main/src/redux/slices/editingV2Project.ts#L56